src/core/DataFile.cpp

```diff
@@ -1638,6 +1639,51 @@ void DataFile::upgrade_1_3_0()
 	}
 }
 
+void DataFile::upgrade_noHiddenAutomationTracks()
+{
+	// convert global automation tracks to non-hidden
+	QDomElement song = firstChildElement("lmms-project")
+		.firstChildElement("song");
+	QDomElement tc = song.firstChildElement("trackcontainer");
+	QDomElement gaTrack = song.firstChildElement("track");
+	if (!gaTrack.isNull()
+		&& gaTrack.attribute("type").toInt() == static_cast<int>(Track::Type::HiddenAutomation))
+	{
+		// global automationclips
+		QDomNodeList aps = gaTrack.elementsByTagName("automationclip");
```

Comment:
IMO the code would become better readable if that was called `automationClips` instead of `aps`. What does "aps" stand for?

Reply:
I think `ap` stands for `AutomationPattern` which is `AutomationClip` now. I agree on renaming.

Reply:
Renamed variables.